### PR TITLE
Dependencies update and syntax improved

### PR DESCRIPTION
**Pull request by enpitsuLin**
The first thing is to update the dependencies

As the dependencies were updated which were be, the package was [Pure ESM](https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c) now, so the syntax needs to be improved, so it will be major version update if merged

### Motivation

In fact, most of the other officially maintained utilities of hast such as [hast-util-sanitize](https://github.com/syntax-tree/hast-util-sanitize) are already Pure ESM, so it's always not bad idea to keep up with the official footsteps, right?

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
-var ClassList = require('./')
-var hastscript = require('hastscript')
-var test = require('tape')
+import ClassList from './index.js'
+import { h as hastscript } from 'hastscript'
```
Comment (enpitsuLin) [inline test.js:2]: here is breaking change in `hastscript` since [7.0.0](https://github.com/syntax-tree/hastscript/releases/tag/7.0.0) which also use ESM

**Comment by brechtcs** [comment]
Thanks, LGTM at first sight. I'll have a more detailed look later when I have more time before merging.

I guess with the ESM migration it'll probably require a major version bump when released? (No need to update the PR to reflect that though, I'll take care of that later.)

File: test.js (hunk 1 above)
Comment (brechtcs) [reply test.js:2]: Should be okay, it's only a devDependency, and I think I'll probably have to do a major version bump anyway.

**Comment by brechtcs** [comment]
Done. Thanks for the contribution!

You mentioned you had some other things in mind to change. Unless you think those might still take a while to finish, I might wait for those before releasing.

**Comment by enpitsuLin** [comment]
There are really some other jobs that have been done here and I will open PR later

**Pull request merged**